src/components/organisms/ReviewTemplate/ReviewTemplate.tsx

```diff
@@ -1,40 +1,75 @@
+import { TemplateType, TemplateStatus } from '@utils/enum';
+import concatClassNames from '@utils/concat-class-names';
 import styles from './ReviewTemplate.module.scss';
+import { ReviewTemplateProps } from './review-template.types';
+
+const templateTypeDisplayMappings = (type: TemplateType) =>
+  ({
+    [TemplateType.NHS_APP]: 'NHS App',
+    [TemplateType.SMS]: 'Text message (SMS)',
+    [TemplateType.EMAIL]: 'Email',
+    [TemplateType.LETTER]: 'Letter',
+  })[type];
+
+const templateStatustoDisplayMappings = (status: TemplateStatus) =>
+  ({
+    [TemplateStatus.NOT_YET_SUBMITTED]: 'Not yet submitted',
+    [TemplateStatus.SUBMITTED]: 'Submitted',
+  })[status];
 
 export function ReviewTemplate({
   form,
   ...props
 }: Readonly<ReviewTemplateProps>) {
   return (
     <>
+      <div className='notify-confirmation-panel nhsuk-heading-l'>
+        {props.sectionHeading}
+      </div>
       <ZodErrorSummary errorHeading={form.errorHeading} state={form.state} />
       <h1
         data-testid='preview-message__heading'
         className={styles.review__heading}
       >
-        <span
-          data-testid='preview-message__heading-caption'
-          className='nhsuk-caption-l'
-        >
-          {props.sectionHeading}
-        </span>
-        {props.templateName}
+        {props.template.name}
       </h1>
-      <Details data-testid='preview-message-details'>
-        <Details.Summary data-testid='preview-message-details__heading'>
-          {props.details.heading}
-        </Details.Summary>
-        <Details.Text data-testid='preview-message-details__text'>
-          {props.details.text.map((val) => (
-            <p key={val.id}>{val.text}</p>
-          ))}
-        </Details.Text>
-      </Details>
+      <Container
+        className={concatClassNames('nhsuk-u-margin-bottom-4', 'nhsuk-body-m')}
+      >
+        <div className={styles.preview}>
+          <Row className={styles.preview__row}>
+            <Col width='one-third' className={styles.preview__col}>
+              <div className={styles.preview__col_heading}>Template ID</div>
+            </Col>
+            <Col width='two-thirds' className={styles.col}>
+              {props.template.id}
+            </Col>
+          </Row>
+          <Row className={styles.preview__row}>
+            <Col width='one-third' className={styles.preview__col}>
+              <div className={styles.preview__col_heading}>Type</div>
+            </Col>
+            <Col width='two-thirds' className={styles.col}>
+              {templateTypeDisplayMappings(props.template.templateType)}
+            </Col>
+          </Row>
+          <Row className={styles.preview__row}>
+            <Col width='one-third' className={styles.preview__col}>
+              <div className={styles.preview__col_heading}>Status</div>
+            </Col>
+            <Col width='two-thirds' className={styles.col}>
+              <div className='nhsuk-tag'>
```

Comment:
Same here there is a Tag component https://main--65aa76b29d00a047fe683b95.chromatic.com/?path=/docs/content-presentation-tag--docs

Reply:
updated